documentation/config_file_puppetserver.md

```diff
@@ -30,6 +30,7 @@ The `puppetserver.conf` file contains settings for Puppet Server software. For a
     JRuby flushing can be useful for working around buggy module code that would otherwise cause memory leaks, but it slightly reduces performance whenever a new JRuby instance reloads all of the Puppet Ruby code. If memory leaks from module code are not an issue in your deployment, the default value of 0 performs best.
     * `borrow-timeout`: Optional. The timeout in milliseconds, when attempting to borrow an instance from the JRuby pool. The default is 1200000.
     * `use-legacy-auth-conf`: Optional. The method to be used for authorizing access to the HTTP endpoints served by the "master" service. The applicable endpoints are listed in [Puppet v3 HTTP API](/puppet/latest/reference/http_api/http_api_index.html#puppet-v3-http-api).
+    * `compile-mode`: Optional, experimental.  Used to control JRuby's "CompileMode", which may improve performance.  The default value is `:off`, which is the most conservative value.  A value of `:jit` will enable JRuby's "just in time" compilation of Ruby code.  A value of `:force` will cause JRuby to attempt to pre-compile all Ruby code ahead of time.
```

Comment:
Crap, good call, thanks.  Will fix.  I wonder if there's a not-insane way to add test coverage for that... :(